aea/cli/utils/package_utils.py

```diff
@@ -283,58 +282,6 @@ def find_item_locally(
     return package_path, item_configuration
 
 
-def find_item_in_distribution(  # pylint: disable=unused-argument
```

Comment:
Please keep until we know for sure we don't need it anymore. Also keep in other places for now imo.

Reply:
OK